mcp_server.py

```
@@ -0,0 +1,50 @@
+from mcp.server.fastmcp import FastMCP
+
+from src.models.financial_models import AssetClass, Equity
+
+# Initialize the MCP server
+mcp = FastMCP("DashFin-Relationship-Manager")
+
+# Global instance of the graph logic
+graph = AssetRelationshipGraph()
```

Ruff (0.14.10)

Undefined name `AssetRelationshipGraph`

(F821)